swagger/v1/api_v1.json

```diff
@@ -1195,6 +1195,72 @@
           }
         }
       }
+    },
+    "/api/podcast_episodes": {
```

Comment:
@lboogie04 I'm not sure how this happened but the /api/podcast_episodes endpoint got added to your json file and now theres a duplicate /api/podcast_episodes key causing the final json to error out.

<img width="1920" alt="Screenshot 2023-01-16 at 11 00 55" src="https://user-images.githubusercontent.com/2786819/212638328-6369e13e-fcc3-40ad-9a51-611c8a8694e8.png">